libs/rtemodel/include/RteGenerator.h

```diff
@@ -111,6 +117,20 @@ class RteGenerator : public RteItem
   */
   const std::string& GetWorkingDir() const { return GetItemValue("workingDir"); }
 
+  /**
+   * @brief get the absolute path to the executable with expanded key sequences for specified target
+   * @param target pointer to RteTarget
+   * @return expanded executable
+  */
+  std::string GetExecutable(RteTarget* target) const;
+
+  /**
+   * @brief get the absolute paths to executables for all host types with the expanded key sequences for specified target
+   * @param target pointer to RteTarget
+   * @return map with host type ("win", "linux", "mac", "other" or "all") as key and path to executable as value
+  */
+  std::map<std::string, std::string> GetExecutables(RteTarget* target) const;
```

Comment:
I still do not understand why all commands are needed in a separate variable? Where should this method be used?

Reply:
`RteGenerator::GetExecutables()` is used from `ProjMgrYamlCbuild::SetGeneratorsNode()`. I'm not sure what you suggest that we use instead, but to me it made sense to have a method providing the executable files of a generator.

Reply:
It kind of fulfills the same purpose as `RteGenerator::GetExpandedCommandLine()`. That function didn't fulfill my needs, so I wanted the same information but in a more usable form than a text blob.